[2024-12-11T21:26:09.030+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T21:26:09.067+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-11-15T00:00:00+00:00 [queued]>
[2024-12-11T21:26:09.082+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-11-15T00:00:00+00:00 [queued]>
[2024-12-11T21:26:09.087+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-11T21:26:09.115+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_book_data> on 2024-11-15 00:00:00+00:00
[2024-12-11T21:26:09.129+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2101) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-11T21:26:09.132+0000] {standard_task_runner.py:72} INFO - Started process 2123 to run task
[2024-12-11T21:26:09.151+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_google_books', 'fetch_book_data', 'scheduled__2024-11-15T00:00:00+00:00', '--job-id', '1122', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpyf6nm7ti']
[2024-12-11T21:26:09.161+0000] {standard_task_runner.py:105} INFO - Job 1122: Subtask fetch_book_data
[2024-12-11T21:26:09.299+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-11-15T00:00:00+00:00 [running]> on host bb74d1700244
[2024-12-11T21:26:09.447+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_google_books' AIRFLOW_CTX_TASK_ID='fetch_book_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-15T00:00:00+00:00'
[2024-12-11T21:26:09.451+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-11T21:26:09.452+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-11T21:26:09.453+0000] {logging_mixin.py:190} INFO - Current task name:fetch_book_data state:running start_date:2024-12-11 21:26:09.068643+00:00
[2024-12-11T21:26:09.454+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books and current dag run status:running
[2024-12-11T21:26:09.455+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T21:26:18.179+0000] {logging_mixin.py:190} INFO - Failed to fetch data. Status code: 429
[2024-12-11T21:26:18.213+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T21:26:18.228+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T21:26:18.229+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_google_books, task_id=fetch_book_data, run_id=scheduled__2024-11-15T00:00:00+00:00, execution_date=20241115T000000, start_date=20241211T212609, end_date=20241211T212618
[2024-12-11T21:26:18.244+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-11T21:26:18.245+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-11T21:26:18.247+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books queued_at:2024-12-11 21:26:06.231138+00:00
[2024-12-11T21:26:18.248+0000] {logging_mixin.py:190} INFO - Task hostname:bb74d1700244 operator:PythonOperator
[2024-12-11T21:26:18.291+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T21:26:18.335+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T21:26:18.340+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
